src/main/java/com/wcc/platform/domain/cms/attributes/Event.java

```diff
@@ -0,0 +1,9 @@
+package com.wcc.platform.domain.cms.attributes;
+
+import java.util.List;
+
+public record Event (String topic, Category category, String startDate, String endDate, String title, String speaker,
+                     String description, Image image, SimpleLink simpleLink, List<EventResource> eventResources ){
+
```

Comment:
you need to enable autoformat, I dont think this format is correct in your IDEA.

Reply:
done